CONTRIBUTING.md

```diff
@@ -608,18 +608,34 @@ We use [Cypress](https://www.cypress.io/) for integration tests. Tests can be ru
 ```bash
 export SUPERSET_CONFIG=tests.superset_test_config
 export CYPRESS_BASE_URL="http://localhost:8081"
+# ENABLE_REACT_CRUD_VIEWS should be enabled for Cypress testing
+sed -i '/ENABLE_REACT_CRUD_VIEWS = False/c\ENABLE_REACT_CRUD_VIEWS = True' superset/config.py
+sed -i '/ENABLE_REACT_CRUD_VIEWS = os.environ.get("ENABLE_REACT_CRUD_VIEWS", False)/c\ENABLE_REACT_CRUD_VIEWS = True' tests/superset_test_config.py
+superset drop_db --yes
+superset create_db
 superset db upgrade
 superset init
 superset load_test_users
 superset load_examples
 superset run --port 8081
 ```
 
-Run Cypress tests:
+Build Frontend:
 
 ```bash
 cd superset-frontend
 npm run build
+```
+
+Alternatively you can proceed with Superset and Frontend running in Docker (hot reload for frontend not supported yet):
+
+```bash
+SUPERSET_CYPRESS_CONFIG=yes SUPERSET_CONFIG=tests.superset_test_config docker-compose up
```

Comment:
isn't `SUPERSET_CONFIG=tests.superset_test_config` already set in the instructions on line 609?

Reply:
In 609 instruction it is said how to run everything without docker usage - I added this to be clear in required steps.

Reply:
ok, I see.. It seems to me that just passing in the cypress env `SUPERSET_CYPRESS_CONFIG=yes` should be all that you need to run docker for cypress.. can we move `SUPERSET_CONFIG` to the `with_cypress` method instead?